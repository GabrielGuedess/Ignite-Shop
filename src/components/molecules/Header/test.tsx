import { render } from '@testing-library/react';

import { Header } from '.';

describe('<Header />', () => {
  it('should render the header correctly', () => {
    const { container } = render(<Header />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
