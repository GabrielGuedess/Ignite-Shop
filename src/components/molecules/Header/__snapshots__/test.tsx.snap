// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the header correctly 1`] = `
<header
  class="c-iBtCWD"
>
  <img
    alt="Logo"
    data-nimg="1"
    decoding="async"
    height="52"
    loading="lazy"
    src="/svg/logo.svg"
    style="color: transparent;"
    width="130"
  />
</header>
`;
